### ISSUE TITLE
Unmet peer dependency when installing 0.6.0

### ISSUE THREAD

**Issue by kirjs** [akiran/react-highlight#12]
```
└─┬ react-highlight@0.6.0 
  ├── UNMET PEER DEPENDENCY jasmine-core@*
  ├── UNMET PEER DEPENDENCY karma@>=0.9
  ├── karma-jasmine@0.3.6 
  ├── karma-phantomjs-launcher@0.2.1 
  └── UNMET PEER DEPENDENCY phantomjs@>=1.9
```

**Comment by kirjs** [comment]
Also not sure why `jasmine-core` and `karma` are not dev dependencies.

**Comment by kirjs** [comment]
Fixed in 0.6.1
